style/row.less

```diff
@@ -1,11 +1,13 @@
 .rdg-row {
   contain: strict;
   contain: size layout style paint;
-  display: flex;
```

Comment:
We know that flex+sticky position cells can be buggy in Firefox, but I wonder if grid+sticky works fine or not.